build failed. extra semi-colon

mgconsole/src/utils/future.hpp +276

builds fine after the semi-colon is removed

Hello @antejavor 

Thank you for the quick response. 

gcc -v
gcc version 10.2.1 20210110 (Debian 10.2.1-6) 

mkdir build; cd build; cmake ..;make

[ 94%] Building CXX object src/CMakeFiles/mgconsole.dir/batch_import.cpp.o
In file included from /mgconsole/src/batch_import.cpp:26:
/mgconsole/src/utils/future.hpp:276:2: error: extra ‘;’ [-Werror=pedantic]
  276 | };  // namespace utils

--- a/src/utils/future.hpp
+++ b/src/utils/future.hpp
@@ -273,4 +273,4 @@ std::pair<Future<T>, Promise<T>> FuturePromisePairWithNotifications(std::functio
   return std::make_pair(std::move(future), std::move(promise));
 }
 
-};  // namespace utils
+}  // namespace utils

